lib/sqlalchemy/sql/selectable.py

```diff
@@ -3527,7 +3554,7 @@ def scalar_subquery(self) -> ScalarSelect[Any]:
         if self._label_style is not LABEL_STYLE_NONE:
             self = self.set_label_style(LABEL_STYLE_NONE)
 
```

Comment:
**mike bayer** (zzzeek) wrote:

emove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325